### DISCUSSION TITLE
Stupid question: Is Termux still maintained ?

### DISCUSSION THREAD

**Discussion by freebrowser1** [termux/termux-app#2804] [General]
In 2021 lots of new versions appeared and after version 118 in stopped on Jan 7.

Is this still maintained ?
I am still happy with version 118, but I worry about that it won't be maintained anymore ?

**Comment by ghost** [comment]
Yes it's still maintained, and that was just updated a few months ago (it's not a long time). there's a rule to remember that "If it ain't broke, don't touch it".

It's also good to mention that we also update and add packages as we provision it to users to keep termux packages up-to-date and not just the app. upcoming version is v0.119 but not yet planned for release.

Also you can also check the commit date of the termux app and packages repository to actually see what's going on to the source code!

**Reply by freebrowser1** [reply]
Thanks ! And it it not broken for me, it works flawlessly.

**Reply by creepy-pasta101** [reply]
Also, note that you're free to check out the debug builds of the app from github actions.

**Reply by sabretooth75** [reply]
I need a link to download termux and its plugins, especially the GuI that won't install from fdroid.thanks

**Comment by luisdavim** [comment]
There are a lot of stale PRs, some have nice changes that I'd like to see merged. 

Could the maintainers of this project go through the existing PRs and ask for rebases on the one's they want to use and close the ones they don't?
